xml/security_ldap.xml

```diff
@@ -389,290 +387,57 @@ MAY (userPassword $ searchGuide $ seeAlso $ businessCategory <co xml:id="co.ldap
-   Kerberos Client</guimenu> </menuchoice>.
+   The &ds389; is controlled by 3 primary commands:
   </para>
-  <figure xml:id="fig.yast2.ldapkerberos.ldap">
-   <title><guimenu>LDAP and Kerberos Client</guimenu> Window</title>
-   <mediaobject>
-    <imageobject role="html">
-     <imagedata fileref="yast2_auth_client_config.png" width="65%" format="PNG"/>
-    </imageobject>
-    <imageobject role="fo">
-     <imagedata fileref="yast2_auth_client_config.png" width="65%" format="PNG"/>
-    </imageobject>
-   </mediaobject>
-  </figure>
-
-  <para>
-   To configure an LDAP client, follow the procedure below:
-  </para>
-  <procedure xml:id="pro.security.auth.ldap">
-   <step>
-    <para>
-     In the window <guimenu>LDAP and Kerberos Client</guimenu>, click
-     <guimenu>Change Settings</guimenu>.
-    </para>
-    <para>
-     Make sure that the tab <guimenu>Use a Directory as Identity Provider
-     (LDAP)</guimenu> is chosen.
-    </para>
-    <informalfigure>
-     <mediaobject>
-      <imageobject role="html">
-       <imagedata fileref="yast2_auth_client_ldap.png" width="65%" format="PNG"/>
-      </imageobject>
-      <imageobject role="fo">
-       <imagedata fileref="yast2_auth_client_ldap.png" width="65%" format="PNG"/>
-      </imageobject>
-     </mediaobject>
-    </informalfigure>
-   </step>
-   <step>
-    <para>
-     Specify one or more LDAP server URLs, host names, or IP addresses under
-     <guimenu>Enter LDAP server locations</guimenu>. When specifying multiple
-     addresses, separate them with spaces.
-    </para>
-   </step>
-   <step>
-    <para>
-     Specify the appropriate LDAP distinguished name (DN) under
-     <guimenu>DN of Search Base</guimenu>. For example, a valid entry could
-     be <literal>dc=example,dc=com</literal>.
-    </para>
-   </step>
-   <step>
-    <para>
-     If your LDAP server supports TLS encryption, choose the appropriate
-     security option under <guimenu>Secure LDAP Connection</guimenu>.
-    </para>
-    <para>
-     To first ask the server whether it supports TLS encryption and be able
-     to downgrade to an unencrypted connection if it does not, use
-     <guimenu>Secure Communication via StartTLS</guimenu>.
-    </para>
-   </step>
-   <step>
-    <para>
-     Activate other options as necessary:
-    </para>
-    <itemizedlist>
+  <variablelist>
+   <varlistentry>
+    <term><command>dsctl</command></term>
+    <listitem>
+     <para>
+      Manages a local instance and requires &rootuser; permissions. Used for
+      starting, stopping, backing up the database and more.</para>
+    </listitem>
+   </varlistentry>
+    <varlistentry>
+     <term><command>dsconfig</command></term>
      <listitem>
       <para>
-       You can <guimenu>Allow users to authenticate via LDAP</guimenu> and
-       <guimenu>Automatically Create Home Directories</guimenu> on the local
-       computer for them.
+       Manages a remote or local instance configuration. Requires
```

Comment:
I think this is a bit unclear. The best way is to say "manages an instances configuration via it's external interfaces. This allows you to make configuration changes remotely on the instance"

Reply:
Added.